pkg/config/config_template.yaml

```diff
@@ -733,6 +733,12 @@ api_key:
 #
 # enable_gohai: true
 
+## @param enable_signing_metadata - boolean - optional - default: true
+## @env DD_ENABLE_SIGNING_METADATA - boolean - optional - default: true
+## Enable the Linux package signing medatada collection.
+#
+# enable_signing_metadata: true
```

Comment:
Soemwhat of a naming nit but this is rather ambiguous as it can mean both "enable signing of some metadata" and "collection of some signing metadata". Could we use `enable_signing_metadata_collection` instead (or something similar) for clarity?